### ISSUE TITLE
HardInfo could use a new logo

### ISSUE THREAD

**Issue by bp0** [lpereira/hardinfo#388]
![Screenshot from 2019-06-30 16-18-59](https://user-images.githubusercontent.com/1758090/60402259-ca969300-9b52-11e9-974e-0478646f2411.png)

The old low-resolution logo looks pretty bad among newer ones.

**Comment by lpereira** [comment]
Yeah, having a SVG version of the icon would be nice.  That reminded me: need to get some stickers for the 0.6 release :)

**Comment by bp0** [comment]
Some SVG art that is free to use with attribution.
Maybe use one of each of these sets, combined to remake the older icon:
* https://www.flaticon.com/search?word=motherboard
* https://www.flaticon.com/search?word=caliper

I might give it a shot later.

**Comment by bp0** [comment]
Just as an example, this is not the one.
![hardinfoiconex](https://user-images.githubusercontent.com/1758090/60404772-a4342000-9b71-11e9-8046-00871627b0d8.png)

**Comment by lpereira** [comment]
Looking for HardInfo SVG lead me to [this address](https://commons.wikimedia.org/wiki/File:Faenza-hardinfo.svg), containing this image:

![Faenza-hardinfo svg](https://user-images.githubusercontent.com/15001/60555168-4712a880-9cf0-11e9-9815-db8452fcb881.png)

This icon looks very nice, and seem to derive from [this set](https://code.google.com/archive/p/faenza-icon-theme/downloads).  I like the caliper, so I can maybe redraw it someday.  Making the pixel version wasn't that complicated, but I'm not really good at drawing vectors. :)

**Comment by bp0** [comment]
It's not really a logo, but a good icon. Certainly much better than the icon that is now.
